### ISSUE TITLE
[SUBMIT] Setting up a Nuxt app with Strapi Using Graphql

### ISSUE THREAD

**Issue by uma-victor1** [strapi/community-content#469]
**My resource**

**My content is**
Hello 👋 
I would like to write an article on setting up a Nuxt app with Strapi using Graphql

## Goal
In this tutorial, we will look at how we can connect the strapi backend to our Nuxt frontend and fetch some data using Graphql. We’ll also look at how to setup and configure such jamstack project.

## Outline
- The need for headless CMS
    - Why use strapi?
    - How it all works
- setting up the Nuxt frontend
    - Configuring Nuxt to work with Apollo Client
- Creating a Strapi Collection Type
- Install the Strapi GraphQL Plugin
- Using the GraphQL Playground
- Adding a query to our Nuxt client
- Rendering Strapi data in the Nuxt client

## Resources
- Github code
- Resources

## Conclusion
// conclusion

- [x] A Strapi Tutorial / Guide / How to article
- [x] About Headless, CMS, javascript, open source, or jamstack
- [ ] Other, specify

**What do you all think?**
❤️ I love it!
🚀 I can help you!

Thank you for submitting your article proposal.

**Comment by ShadaW11** [comment]
Hello @uma-victor1 can you please let me know what kind of application will you be creating? what will the app do? and what are the features? thank you for submitting the outline.

**Comment by uma-victor1** [comment]
Hello @ShadaW11, I will be building an event app. The app will display the events that we get from strapi API. It will have search functionality and users can select an event based on the type of event.

**Comment by ShadaW11** [comment]
Hello @uma-victor1 Could you please add the features of the app within the outline. Here is a intro + sample outline:

My resource
Svelte is a component framework that runs at build time, converting your components into highly efficient imperative code that surgically updates the DOM. Svelte approach to building web apps is unique and I will like to teach the Strapi community how they can build a to-do app with Svelte using Strapi as the CMS and Tailwind for styling.

Outline:

Introduction
Goals
Prerequisites
Creating a Strapi Project
Creating Content Types
Setting up Roles and Permissions
Bootstrap a Svelte Project
Adding TailwindCSS to our project
Fetch Todo items
Authentication and Authorization
Create todo items
Update todo items
Delete todo items
Conclusion

**Comment by ShadaW11** [comment]
Hello @uma-victor1 any update?

**Comment by uma-victor1** [comment]
Hi @ShadaW11, here is the updated outline


## Outline 
- The need for headless CMS
    - Why use strapi?
    - How it all works
- setting up the Nuxt frontend
    - Configuring Nuxt to work with Apollo Client
- Creating a Strapi Collection Type
- Install the Strapi GraphQL Plugin
- Using the GraphQL Playground
- Adding a query to our Nuxt client
- Rendering Strapi data in the Nuxt client
- Implementing Top Navigation
- Displaying our events
- Filtering our Events
- Searching for Events
- Wrap Up

**Comment by ShadaW11** [comment]
@uma-victor1 So you will be creating an event app with Nuxt and strapi?

**Comment by ShadaW11** [comment]
Hello @uma-victor1 when can you deliver the article?

**Comment by uma-victor1** [comment]
Before the 20th of this month

**Comment by ShadaW11** [comment]
Can you please confirm that you will update the tutorial after the v4 release.
You will get paid once the tutorial is on the blog.
The additional article update after V4 is not update, can you confirm that this is okay for you? If yes, please reply and go ahead and get started. Thank you.

**Comment by uma-victor1** [comment]
@ShadaW11 Yeah. This is okay by me. I will update the tutorial once the v4 is released.

**Comment by ShadaW11** [comment]
Great please start working on the tutorial. @uma-victor1

**Comment by ShadaW11** [comment]
Please review the guidelines on how to write the article, where to submit and all other details in the FAQ section in this page https://strapi.io/write-for-the-community

**Comment by uma-victor1** [comment]
Hi @ShadaW11 

I am done with the article.

Here is [the final draft](https://www.dropbox.com/scl/fi/nemnt0s8ai9gobi5pnnq5/SP-Setting-up-a-Nuxt-app-with-Strapi-Using-Graphql-Building-an-Event-App.paper?dl=0&rlkey=x6rhpjk4oyyxa5bw1howyi6z9):

Regards.

**Comment by ShadaW11** [comment]
@uma-victor1 our reviewer left feedback on your article. please check. thank you

**Comment by uma-victor1** [comment]
@ShadaW11 I'll check it out thanks.
